### PR TITLE
KFSPTS-13785 clean up spring bean overrides for PayeeACHAccount

### PR DESCRIPTION
**Pull request by db483**

### CODE REVIEW

File: src/main/resources/edu/cornell/kfs/pdp/businessobject/datadictionary/PayeeACHAccount.xml [hunk 1]
```diff
@@ -89,254 +38,93 @@
...
+          <property name="propertyName" value="lookupFields" />
+          <property name="propertyNameForElementCompare" value="attributeName" />
+          <property name="element">
+            <bean parent="FieldDefinition" p:attributeName="bankAccountTypeCode" />
+          </property>
+          <property name="insertAfter">
+            <list>
+              <bean parent="FieldDefinition" p:attributeName="payeeEmailAddress" />
+              <bean parent="FieldDefinition" p:attributeName="payeeName" />
+              <bean parent="FieldDefinition" p:attributeName="payeePrincipalName" />
```
Comment (jhulslander) [inline src/main/resources/edu/cornell/kfs/pdp/businessobject/datadictionary/PayeeACHAccount.xml]: I'm a little confused, based on the diff, payeePrincipalName is our only custom field.  Are payeeEmailAddress and payeeName already part of bse code's lookup fields?

Comment (db483) [reply src/main/resources/edu/cornell/kfs/pdp/businessobject/datadictionary/PayeeACHAccount.xml]: Good question. payeePrincipalName is our custom field but we also customized the lookup to add other existing fields to the search fields.

Comment (db483) [reply src/main/resources/edu/cornell/kfs/pdp/businessobject/datadictionary/PayeeACHAccount.xml]: Basically I tried keeping the lookup and inquiry looking the same after cleaning up this file to use overrides.

Comment (jhulslander) [reply src/main/resources/edu/cornell/kfs/pdp/businessobject/datadictionary/PayeeACHAccount.xml]: ah, thank you, this should be fine

**Comment by jhulslander** [comment]
This should be fine to merge once the unit tests complete

**Pull request merged**